app/mailers/user_mailer.rb

```diff
@@ -20,10 +20,12 @@ def admin_message_email(user, comment)
   end
 
   def purge_notification
-    recipients = User.with_any_role(:public_health, :public_health_enroller)
+    recipients = User.with_any_role(:admin)
     @expiration_date = Chronic.parse(ADMIN_OPTIONS['weekly_purge_date']).strftime('%A %B %d, at %l:%M %p %Z')
 
     recipients.each do |user|
+      next if user.jurisdiction&.name == 'USA'
```

Comment:
I think it would be better to include 'USA' users, this way the admin who sent the email will receive the email and know that it was sent out.